tests/foreman/destructive/test_clone.py

```diff
@@ -57,12 +57,12 @@ def test_positive_clone_backup(target_sat, sat_ready_rhel, backup_type, skip_pul
     backup_result = target_sat.cli.Backup.run_backup(
         backup_dir='/var/backup',
         backup_type=backup_type,
-        options={'assumeyes': True, 'plaintext': True, 'skip-pulp-content': skip_pulp_content},
+        options={'assumeyes': True, 'plaintext': True, 'skip-pulp': skip_pulp},
```

Comment:
`skip-pulp-content` option wouldn't change in this case 
```suggestion
        options={'assumeyes': True, 'plaintext': True, 'skip-pulp-content': skip_pulp},
```

Reply:
True, my bad.